config/ImageMagick.rc.cmake

```diff
@@ -10,8 +10,8 @@
 /////////////////////////////////////////////////////////////////////////////
 
 VS_VERSION_INFO VERSIONINFO
- FILEVERSION @MagickLibVersionNumber@
- PRODUCTVERSION @MagickLibVersionNumber@
+ FILEVERSION "@MagickLibVersionNumber@"
```

Comment:
Now that this supports everything maybe we can use the regular ImageMagick.rc instead?

Reply:
Hmm, maybe. The upstream one does not include quotes, though, so that might break.